Gemfile

```diff
@@ -40,7 +40,10 @@ gem 'sass'
 gem 'scss-lint'
 
 #gemas para token
+gem 'haml'  
+gem 'rack-cors', :require => 'rack/cors'
```

Comment:
Are you really need this? https://github.com/lynndylanhurley/devise_token_auth/blob/master/docs/config/cors.md